day2_exercises/fMRI_homework_20151107.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Nov  7 09:02:19 2015
+
+@author: Elissaios
+"""
+
+#%% Load nii image (after it was unzipped)
+import nibabel as nib
+
+folder = r"c:\Anaconda3\Lib\site-packages\nibabel\tests\data";
+#filePath = path + 'ds107_sub001_highres.nii';
+filePath = folder + r"\example_nifti2.nii";
```

Comment:
Consider ``os.path.join`` to join file paths.  This makes sure your script will work on Windows as well as Unix.